webtau-reactjs/src/report/Report.js

```diff
@@ -247,9 +247,13 @@ function fullContainerId(test) {
 }
 
 function shortContainerId(test) {
-    return test.className ?
-        shortenClassName(test.className):
-        shortenFileName(test.fileName)
+    if (test.shortContainerId) {
+        return test.shortContainerId
+    } else if (test.className) {
+        return shortenClassName(test.className)
+    } else {
```

Comment:
will we ever have this case?

Reply:
Shouldn't do.  I had initially left it in while I was working on this change but I think this can go now.